tests/backends/data/test_async_es.py

```diff
@@ -87,24 +89,26 @@ async def test_backends_data_async_es_instantiation_with_settings():
     settings = ESDataBackendSettings(
         ALLOW_YELLOW_STATUS=True,
         CLIENT_OPTIONS={"verify_certs": False, "ca_certs": None},
-        DEFAULT_CHUNK_SIZE=5000,
         DEFAULT_INDEX=ES_TEST_INDEX,
         HOSTS=["https://elasticsearch_hostname:9200"],
         LOCALE_ENCODING="utf-16",
         POINT_IN_TIME_KEEP_ALIVE="5m",
+        READ_CHUNK_SIZE=5000,
         REFRESH_AFTER_WRITE=True,
+        WRITE_CHUNK_SIZE=5000,
```

Comment:
```suggestion
        WRITE_CHUNK_SIZE=4999,
```
(to have a different test value than `READ_CHUNK_SIZE`)

(same for all tests below)

Reply:
Good Idea) Thanks) Updated the write chunk size values)